server/routes/search/searchValidator.ts

```diff
@@ -0,0 +1,67 @@
+import { PrisonerSearchForm } from 'express-session'
+import HmppsError from '../../interfaces/HmppsError'
+import dateOfBirthValidator from '../../utils/dateOfBirthValidator'
+
+export function isAlphabetic(field: string) {
+  return /^[A-Za-z']*$/.test(field)
```

Comment:
can't the search fields contain `*` and `%` too?

Reply:
ah yes - that is wrong - will update it